### PR TITLE
v3 transformers

### PR DESCRIPTION
**Pull request by eitan101**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,13 @@
...
 
-The SDK provides a simple node JS wrapper for the [LivePerson messaging API][1].
+The SDK provides a simple node js wrapper for the [LivePerson messaging API][1].
 
 ## Disclaimer
 A new major version of the SDK will be released soon with a breaking change:
-The current SDK will start sending notifications once connected.
-The next version will require explicit registration.
-Specifically, the function `queryMessages` will be removed and replaced with a new function.
+Current SDK will start sending notifications once connected.
```
Comment (miki2826) [inline README.md:14]: @eitan101 please fetch the latest changes  that were pushed

File: README.md [hunk 2]
```diff
@@ -32,13 +32,13 @@ Specifically, the function `queryMessages` will be removed and replaced with a n
...
-    username: process.env.USERNAME,
-    password: process.env.PASSWORD
+    accountId: process.env.LP_ACCOUNT,
+    username: process.env.LP_USER,
+    password: process.env.LP_PASS
 });
 
 agent.on('connected', () => {
-    console.log(`connected... ${agent.agentId}`);
+    console.log(`connected...`);
```
Comment (miki2826) [inline README.md:41]: This was to demonstrate that you have the id available.

File: lib/AgentSDK.js [hunk 3]
```diff
@@ -60,7 +60,14 @@ class SDK extends Events {
...
                     accountId: conf.accountId
                 }, (err, data) => {
-                    this.agentPID = data && data.pid;
+                    // TODO: Ugly hack, remove it later.
+                    // Currently there is a bug in the ac-users.
+                    // Transoformers adds convDetails.getMyRole() and ms.MessagingEvents.isMe()
+
+                    // Uncomment this line after fixing ac-users
+                    // this.agentId = data && data.pid;
+                    this.agentId = {error: "agentId not supported yet. Use convDetails.getMyRole() and ms.MessagingEvents.isMe()."};
```
Comment (miki2826) [inline lib/AgentSDK.js:69]: adding the function `convDetails.getMyRole()` means it is another breaking change in the next version.
If we actually decide to go with it we should add it to the documentation as well in the disclaimer part

**Review by miki2826 — COMMENTED** [review]
did you run the tests?

**Comment by eitan101** [comment]
I already committed by mistake to the master.
Abi's work was merged automatically.

**Pull request closed without merging**